pkg/controller/job/pod_failure_policy_test.go

```diff
@@ -116,10 +116,59 @@ func TestMatchPodFailurePolicy(t *testing.T) {
 					},
 				},
 			},
-			wantJobTerminationMessage: pointer.String("Container main-container for pod default/mypod failed with exit code 2 matching Terminate rule at index 0"),
-			wantCountFailed:           true,
+			wantJobFailureMessage: pointer.String("Container main-container for pod default/mypod failed with exit code 2 matching FailJob rule at index 0"),
+			wantCountFailed:       true,
 		},
-		"second terminate rule matched for exit codes": {
+		"successful containers are skipped by the rules": {
```

Comment:
Add one case for NotIn with values `0` that matches a pod with two containers with exit code 0 and 1

Reply:
added